apps/docs/.storybook/main.ts

```diff
@@ -17,11 +17,11 @@ const config = {
   ],
   addons: [
     getAbsolutePath('@storybook/addon-essentials'),
+    getAbsolutePath('@storybook/addon-actions'),
```

Comment:
This resolves this console warning: 
<img width="1172" alt="Screenshot 2024-03-25 at 8 57 40 AM" src="https://github.com/graveflex/nextjs-vercel/assets/32271860/f1b44418-eb77-4711-b2c8-e9541ecac562">